app/services/google_service.rb

```diff
@@ -1,7 +1,6 @@
 class GoogleService
-
   def self.decode(token)
-    response = Faraday.new(url: "https://oauth2.googleapis.com").get("/tokeninfo?id_token=#{token}")
+    response = Faraday.new(url: 'https://oauth2.googleapis.com').get("/tokeninfo?id_token=#{token}")
```

Comment:
could probably use an environment variable for all urls in a future refactor, in case something changes with our apis and me need to easily change things across the app. just a thought for future dev.